kroxylicious-operator/src/main/resources/META-INF/fabric8/filters.filter.kroxylicious.io-v1.yml

```diff
@@ -50,6 +53,15 @@ spec:
                   description: The filter's configuration.
                   type: object
                   x-kubernetes-preserve-unknown-fields: true
+                configTemplate:
+                  description: The filter's configuration, subject to config provider interpolation.
+                  type: object
+                  x-kubernetes-preserve-unknown-fields: true
+              oneOf:
+                - required:
+                    - config
```

Comment:
Do we want to keep `config`, or just have `configTemplate`. I think just moving to `configTemplate` is simplest. The doesn't seem much value in having plain config.

Reply:
> I think just moving to configTemplate is simplest.
+1

Reply:
+1 to configTemplate only